dstk-infra/postgres/patches/20231226_current-model-version-key.sql

```diff
@@ -0,0 +1,7 @@
+\connect dstk;
+
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
```

Comment:
This isn't needed

Reply:
Just the `CREATE EXTENSION...` bit?

Reply:
Yeah, that's an only-needs-to-happen-once but that gets taken care of when the db is very first created. It gets ignored on subsequent runs, so no need to have it in here

Reply:
Makes sense, thank you for the explanation!